src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -114,7 +115,17 @@ public void Start()
                 return;
             }
 
-            var incomingMessage = brokeredMessageConverter.Convert(message);
+            IncomingMessageDetails incomingMessage;
+            try
+            {
+                incomingMessage = brokeredMessageConverter.Convert(message);
+            }
+            catch (ConfigurationErrorsException exception)
```

Comment:
We should use a different exception type here, ConfigurationErrorsException is for mistakes in the app.config